pepper-apis/config/squid.conf

```diff
@@ -0,0 +1,22 @@
+# turn off caching
+cache deny all
+cache_dir null /tmp
+
+acl SSL_ports port 443
+# elasticsearch port
+acl Safe_ports port 9243
```

Comment:
Should this be listed under the `SSL_ports`? It doesn't look like ElasticSearch uses port `443`.

Reply:
Yes! In fact, I reworked it so it's more locked down and more readable.

Reply:
I saw! Looks great!